modules/administration/pages/actions.adoc

```diff
@@ -117,9 +120,47 @@ Wildcards can be used to run commands across any number of systems.
 Always take extra care to check your commands before issuing them.
 ====
 
+Before you begin, ensure your client is subscribed to the appropriate tools child channel for its installed operating system.
+For more information about subscribing to software channels, see xref:client-configuration:channels.adoc[].
+
+
+// This seems really hacky. Is it still correct? --LKB 2020-07-28
```

Comment:
I think when client is registered with the server, these packages should be installed from the server (while bootstrapping or later).  With the activation key, with the Web UI, or API.

Reply:
tradidional clients: requires extra packages and you need to explicit enable remote-command execution.

salt clients: everything is there and possible as soon as you have salt-minion installed.